CMakeLists.txt

```diff
@@ -7,8 +7,7 @@ project(adobe_source_libraries CXX)
 
 set(CMAKE_CXX_EXTENSIONS OFF)
 if(NOT DEFINED CMAKE_CXX_STANDARD)
-    set(CMAKE_CXX_STANDARD 17)
-    set(CMAKE_XCODE_ATTRIBUTE_CLANG_CXX_LANGUAGE_STANDARD "c++17")
+    set(CMAKE_CXX_STANDARD 20)
```

Comment:
I'm going to assume that dropping the Xcode setting is intentional and correct.

Reply:
Yes. It isn't needed.